spec/system/admin/products_v3/update_spec.rb

```diff
@@ -0,0 +1,700 @@
+# frozen_string_literal: true
+
+require "system_helper"
+
+RSpec.describe 'As an enterprise user, I can manage my products', feature: :admin_style_v3 do
```

Comment:
It's trivial, but seems like a good idea to give this a more specific name:

```suggestion
RSpec.describe 'As an enterprise user, I can update my products', feature: :admin_style_v3 do
```